docs/rst/user_guide/atmosphere/intro.rst

```diff
@@ -37,83 +36,76 @@ Atmosphere modelling
+with respect to :math:`x` and :math:`y`, and we can rewrite :eq:`radprops` as:
 
 .. math::
+   :label: radprops-onedim
 
-   k_{\mathrm{t}} = k_{\mathrm{s}} + k_{\mathrm{a}}
+   x, y, z \longrightarrow p(z), \varpi(z), \sigma_{\mathrm{t}}(z)
 
-.. math::
+If the participating medium is uniform, then :eq:`radprops` reduces to:
 
-   \varpi = \frac{k_{\mathrm{s}}}{k_{\mathrm{t}}}
-
-where:
+.. math::
 
-* :math:`k_{\mathrm{a}}` is the absorption coefficient :math:`[L^{-1}]` and
-* :math:`k_{\mathrm{s}}` is the scattering coefficient :math:`[L^{-1}]`.
+   x, y, z \longrightarrow p_0, \varpi_0, \sigma_{\mathrm{t}, 0}   
 
-The phase function does not vary from one cell to the other; it is the same
-for the whole atmosphere.
-Only the albedo and extinction coefficient are allowed to vary with space.
+where :math:`p_0`, :math:`\varpi_0`, :math:`\sigma_{\mathrm{t}, 0}`  are constants.
 
 The structure of the participating medium is assumed to be isotropic, *i.e.* the
 properties of the medium embedded in one cell is invariant to rotation of the
 cell.
 
-.. note::
-   So far, only purely molecular atmospheres are supported.
-   Work on adding aerosols to the atmosphere is ongoing.
+Atmospheric constituents
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Atmospheric constituents are divided into two broad categories:
+
+* **molecules** : gas state particles, *e.g.*, H2O, CO2, O3, N2O, CO, CH4, O2 and so on.
+* **particles** : solid or liquid state particles, *e.g.*, water droplets, ice crystals, dust particles, and so on.
+
 
 Atmosphere types
-----------------
-
-Eradiate provides two atmosphere types:
-
-* homogeneous atmosphere
-  (:class:`~eradiate.scenes.atmosphere.HomogeneousAtmosphere`): radiative
-  properties are uniform within the atmosphere.
-* heterogeneous atmosphere
-  (:class:`~eradiate.scenes.atmosphere.HeterogeneousAtmosphereLegacy`): radiative
-  properties are non-uniform within the atmosphere.
-
-.. image:: fig/atmosphere-classes.png
-   :align: center
-
-Both atmosphere types inherit an abstract atmosphere base type
-(:class:`~eradiate.scenes.atmosphere.Atmosphere`),
-parameterised by a top-of-atmosphere altitude (``toa_altitude``) and a width
-(``width``).
-As a result, these parameters can be set for both atmosphere types.
-Both atmosphere types use the
-:ref:`Rayleigh scattering phase function <sec-atmosphere-molecular-scattering>`
-to describe the angular distribution of scattered light.
-Each atmosphere type is further characterised by additional specific parameters.
-For more information, refer to the following guide pages:
-
-* :ref:`Homogeneous atmospheres <sec-atmosphere-homogeneous>`.
-* :ref:`Heterogeneous atmospheres <sec-atmosphere-heterogeneous>`.
+^^^^^^^^^^^^^^^^
+
+Atmosphere types are defined based on radiative properties uniformness
+and nature of atmospheric constituents.
+
+============  ===============================  =================================
+Constituents  Uniform                          Non-uniform
```

Comment:
I thought you taught me to use uniform/non-uniform to qualify the radiative properties corrresponding to homogeneous, heterogeneous participating media, respectively.
So the logic is:
* uniform radiative properties -> homogeneous medium
* non-uniform radiative properties -> heterogeneous medium

In the table, the column headings refer to the radiative properties.
I updated the table so that its clearer.
If you want, I can change them to homogeneous and heterogeneous. But I think we used this logic in both the scientific handbook and the rest of Eradiate's documentation/docstrings.

Reply:
Works for me, let's go with this!